packages/web/src/AdminRoutesWithBackButton.js

```diff
@@ -0,0 +1,183 @@
+import AddNoTeamParticipant from './components/admin_page/hackathon/hackathon_no_team_participants/new_no_team_participant';
+import UrlsTable from './components/admin_page/url_shortener/table';
+import { Routes, Route } from 'react-router-dom';
+import BackBtn from './components/admin_page/back_button';
+
+const adminRoutes = [
+    {
+        path: 'members',
+        element: <RenderMembers />,
+    },
+    {
+        path: 'members/actions',
+        element: <MemberActions />,
+    },
+    {
+        path: 'members/add',
+        element: <AddMember />,
+    },
+    {
+        path: 'jobs',
+        element: <RenderJobs />,
+    },
+    {
+        path: 'jobs/add',
+        element: <AddJobs />,
+    },
+    {
+        path: 'jobs/actions',
+        element: <JobActions />,
+    },
+    {
+        path: 'mentors',
+        element: <RenderMentors />,
+    },
+    {
+        path: 'mentors/actions',
+        element: <MentorsActions />,
+    },
+    {
+        path: 'mentors/add',
+        element: <AddMentors />,
+    },
+    {
+        path: 'jury',
+        element: <RenderJury />,
+    },
+    {
+        path: 'jury/add',
+        element: <AddJury />,
+    },
+    {
+        path: 'jury/actions',
+        element: <JuryActions />,
+    },
+    {
+        path: 'sponsors',
+        element: <RenderSponsors />,
+    },
+    {
+        path: 'sponsors/add',
+        element: <AddSponsors />,
+    },
+    {
+        path: 'sponsors/actions',
+        element: <SponsorsActions />,
+    },
+    {
+        path: 'partners',
+        element: <RenderPartners />,
+    },
+    {
+        path: 'partners/add',
+        element: <AddPartners />,
+    },
+    {
+        path: 'partners/actions',
+        element: <PartnersActions />,
+    },
+    {
+        path: 'hackaton/teams',
+        element: <RenderTeams />,
+    },
+    {
+        path: 'hackaton/teams/actions',
+        element: <TeamActions />,
+    },
+    {
+        path: 'hackaton/teams/add',
+        element: <AddNewTeam />,
+    },
+    {
+        path: 'hackaton/teams/members',
+        element: <RenderTeamMembers />,
+    },
+    {
+        path: 'hackaton/teams/members/add',
+        element: <AddTeamMember />,
+    },
+    {
+        path: 'hackaton/teams/members/actions',
+        element: <TeamMemberActions />,
+    },
+    {
+        path: 'hackaton/noteamparticipants',
+        element: <RenderNoTeamParticipants />,
+    },
+    {
+        path: 'hackathon/noteamparticipants/add',
+        element: <AddNoTeamParticipant />,
+    },
+    {
+        path: 'hackathon/noteamparticipants/actions',
+        element: <NoTeamParticipantsActions />,
+    },
+    {
+        path: 's3',
+        element: <S3Panel />,
+    },
+    {
+        path: 's3/objects',
+        element: <RenderStorageObjects />,
+    },
+    {
+        path: 'shortener',
+        element: <UrlsTable />,
+    },
+    {
+        path: 'fswitches',
+        element: <RenderSwitches />,
+    },
+];
```

Comment:
You should space out your code. I've seen you do this quite a bit. Make sure to insert at least one newline between code blocks.